index.html

```diff
@@ -3570,18 +3570,19 @@ <h2>Hosting, Infrastructure, and Systems</h2>
 			<p>Benefits include:</p>
 			<ul>
 				<li>Efficient, sustainable hosting, infrastructure, and systems can create new marketing opportunities for your business: impressing green-minded or privacy-driven customers.</li>
-				<li> Similarly, reducing client device entry requirements may allow you to expand your services to include those using less advanced client devices.</li>
+				<li>Similarly, reducing client device entry requirements may allow you to expand your services to include those using less advanced client devices.</li>
```

Comment:
I don't really understand this point. 

Hosting, Infrastructure, and Systems is centered on the backend right? Yet here we're talking about client devices, which feels like a frontend concern.